tools/fw-update/rs-fw-update.cpp

```diff
@@ -122,7 +122,7 @@ void list_devices(rs2::context ctx)
 int main(int argc, char** argv) try
 {
 #ifdef BUILD_EASYLOGGINGPP
-    rs2::log_to_console(RS2_LOG_SEVERITY_ERROR);
+    rs2::log_to_console(RS2_LOG_SEVERITY_WARN);
```

Comment:
Does this override `RS2_LOG_LEVEL`?

Reply:
from what I see it does.

Reply:
Do we want it to override?

Reply:
There is some inconsistency , log_to_file takes the env var and log_to_console don't,
I can add it , what do you think?

Reply:
I don't want to complicate things... can we just do:
```
if( ! getenv( "RS2_LOG_LEVEL" ))
    rs2::log_to_console(RS2_LOG_SEVERITY_WARN);
```

Reply:
OK, just note that it will not log to console , it will log to a file.
The default behavior when the env var is set is to log to file

Reply:
Oh, I see what you're saying... :(
OK, let's leave it this way.